static/connectors.json

```diff
@@ -6729,8 +6729,8 @@
   },
   {
     "nameWithOwner": "hariso/conduit-connector-s3",
-    "description": "Conduit connector for AWS S3",
-    "createdAt": "2023-11-30 11:03:20 +0000 UTC",
+    "description": "Conduit connector for Amazon S3",
+    "createdAt": "2024-05-17 12:30:24 +0000 UTC",
     "url": "https://github.com/hariso/conduit-connector-s3",
```

Comment:
@hariso should we add this connector to the ignored list?

Reply:
@lovromazgon yup!